Assets/MRTK/SDK/Features/UX/Scripts/Cursors/FingerCursor.cs

```diff
@@ -59,16 +59,18 @@ protected override void UpdateCursorTransform()
 
                 Vector3 indexFingerPosition;
                 Quaternion indexFingerRotation;
+                // If we are unable to get the hand joint default to the Near Pointer's position and rotation
                 if (!TryGetJoint(TrackedHandJoint.IndexTip, out indexFingerPosition, out indexFingerRotation))
                 {
-                    indexFingerPosition = transform.position;
-                    indexFingerRotation = transform.rotation;
+                    indexFingerPosition = Pointer.Position;
```

Comment:
What's the difference here? Isn't the cursor typically placed slightly in front of the pointer or something, or does the previous code just sit in one place?

Reply:
This change was just to ensure that the cursor was using the Pointer interface for accessing it's position. The previous code makes it seem like it just sits in the same place (which has worked and been non-breaking since we've never had near pointer's enabled for non-articulated hands)